docs/src/components/ColorPalette.js

```diff
@@ -0,0 +1,45 @@
+// @flow strict
+import type { Node } from 'react';
+import { Box, Text } from 'gestalt';
+
+type Props = {|
+  name: string,
+  tokenId: string,
+|};
+
+const ColorPalette = ({ name, tokenId }: Props): Node => {
+  const tokenNumbers = [0, 50, 100, 200, 300, 400, 450, 500, 600, 700, 800, 900];
```

Comment:
@ayeshakmaz is it possible to import the `gestalt-design-tokens` package instead and get the values that way?